src/metaFEMObject.cxx

```diff
@@ -62,17 +62,13 @@ FEMObjectElement::
   delete []m_NodesId;
 }
 
-FEMObjectLoad::FEMObjectLoad()
-{
-}
+FEMObjectLoad::FEMObjectLoad() = default;
 
 FEMObjectLoad::~FEMObjectLoad()
 {
-  for(std::vector<FEMObjectMFCTerm *>::iterator it = this->m_LHS.begin();
-      it != this->m_LHS.end();
-      ++it)
+  for(auto & it : this->m_LHS)
     {
-    delete (*it);
```

Comment:
Interesting.  I've not seen this syntax before, but after digging into it, it seems valid.  Cool!

Reply:
This is from this:
https://clang.llvm.org/extra/clang-tidy/checks/modernize-loop-convert.html